### PR TITLE
[SPARK-28831][DOC][SQL]Document CLEAR CACHE statement in SQL Reference

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Document CLEAR CACHE statement in SQL Reference

### Why are the changes needed?
To complete SQL Reference

### Does this PR introduce any user-facing change?
Yes

After change:
image

### How was this patch tested?
Tested using jykyll build --serve

### CODE REVIEW

**Comment by dilipbiswal** [comment]
@huaxingao Could you please attach a screen-shot of the page ?

File: docs/sql-ref-syntax-aux-cache-clear-cache.md [hunk 1]
```diff
@@ -19,4 +19,16 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Clear the RDD cache associated with a SQLContext.
```
Comment (srowen) [inline docs/sql-ref-syntax-aux-cache-clear-cache.md]: Is it worth noting that this clears _all_ cached RDDs (right)? and it's RDDs and DataFrames? I'd back-tick `SQLContext` for consistency, but might we say `SparkSession` instead?

**Review by srowen — CHANGES_REQUESTED** [review]

File: docs/sql-ref-syntax-aux-cache-clear-cache.md [hunk 2]
```diff
@@ -19,4 +19,16 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Clears all cached RDDs and DataFrames associated with a `SparkSession`.
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-cache-clear-cache.md]: @huaxingao Could you double check on the "with a SparkSession" part ? I thought the cached data is scoped beyond a session ?

File: docs/sql-ref-syntax-aux-cache-clear-cache.md [hunk 3]
```diff
@@ -20,15 +20,19 @@ license: |
 ---
 
 ### Description
-Clears all cached RDDs and DataFrames associated with a `SparkSession`.
+`CLEAR CACHE` removes the entries and associated data from the in-memory and/or on-disk cache for all cached tables  and views.
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-cache-clear-cache.md]: do we have 2 spaces here ?

**Comment by dilipbiswal** [comment]
LGTM
@gatorsmile one question here.. so for this case, examples don't add much value. Its still good to have for completeness, right ?

**Comment by gatorsmile** [comment]
LGTM

Thanks! Merged to master.

**Comment by huaxingao** [comment]
Thanks @dilipbiswal @gatorsmile @srowen

**Pull request closed without merging**